mozilla-services/consvc-shepherd
# [SHEP-94] Update Campaign Overviews to show Campaign Name

## References

[SHEP-94](https://mozilla-hub.atlassian.net/browse/SHEP-94)

## Problem Statement

Campaign Summary does not show campaign name.  It shows Advertiser Name; but advertisers can have multiple campaigns and this causes confusion.

## Proposed Changes

Add new field Campaign Name to Campaign Summaries.

## Verification Steps

A step by step guide of how the PR reviewer can verify that changes are working correctly.

- Go to the [Campaign Summary page](https://shepherd.stage.ads.nonprod.webservices.mozgcp.net/admin/consvc_shepherd/campaignsummary/)
- The first column should be "Name"
- The value of the rows should be the value of the "Products & Budgets" section on the Booster Deal UI page.
<img width="364" alt="Screenshot 2024-10-28 at 11 05 29 AM" src="https://github.com/user-attachments/assets/0a4df768-3be3-4188-82a0-b55b1d288d6d">


## Visuals

If applicable, provide screenshots or videos here with the expected new behavior. If not, please remove this section.
<img width="1118" alt="Screenshot 2024-10-25 at 1 08 11 AM" src="https://github.com/user-attachments/assets/809aa64e-cce3-4dfd-93e3-b183a52d6848">
The "Campaign Summaries" page 

[SHEP-94]: https://mozilla-hub.atlassian.net/browse/SHEP-94?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
